staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -189,7 +189,7 @@ func ValidateCustomResourceDefinitionValidation(customResourceValidation *apiext
 		// if validation passed otherwise, make sure we can actually construct a schema validator from this schema
 		if len(allErrs) == 0 {
 			if _, err := apiservervalidation.NewValidator(customResourceValidation.OpenAPIV3Schema); err != nil {
-				allErrs = append(allErrs, field.Invalid(fldPath.Child("openAPIV3Schema"), "", fmt.Sprintf("error building validator: %v", err)))
+				allErrs = append(allErrs, field.Invalid(fldPath.Child("openAPIV3Schema"), customResourceValidation.OpenAPIV3Schema, fmt.Sprintf("error building validator: %v", err)))
```

Comment:
@carlory, @liggitt meant to leave it as `""`. It was right before. :)

Reply:
recovered